napari/_qt/threading.py

```diff
@@ -682,6 +712,12 @@ def _new_worker_qthread(
     **kwargs
         will be passed to the Worker class on instantiation.
 
+    Returns
+    -------
+    tuple
+        ``(worker, thread)``, where ``worker`` is an instance of ``Worker``,
+        and ``thread`` is an instance of ``QThread``.
```

Comment:
normally this would be:

```rst
worker : WorkerBase
    The created worker.
thread : QThread
    The thread on which the worker is running.

Reply:
oh I wondered!  i didn't know you could do that.  thanks